abel/classes/interstage/plasma_lens/impactx.py

```diff
@@ -64,11 +72,37 @@ def track(self, beam0, savedepth=0, runnable=None, verbose=False):
         
         # run ImpactX
         from abel.wrappers.impactx.impactx_wrapper import run_impactx
-        beam, self.evolution = run_impactx(lattice, beam0, nom_energy=self.nom_energy, verbose=False, runnable=runnable, save_beams=self.use_monitors, space_charge=self.enable_space_charge, csr=self.enable_csr, isr=self.enable_isr)
+        beam, self.evolution = run_impactx(lattice, beam0, nom_energy=self.nom_energy, verbose=False, runnable=runnable, save_beams=self.use_monitors, space_charge=self.enable_space_charge, csr=self.enable_csr, isr=self.enable_isr, isr_on_ref_part=self.isr_on_ref_part)
         
         return super().track(beam, savedepth, runnable, verbose)
     
+
+    # ==================================================
+    @property
+    def isr_on_ref_part(self) -> bool | None:
+        """
+        Whether `ISR will be applied to the reference particle <https://impactx.readthedocs.io/en/latest/usage/python.html#impactx.ImpactX.isr_on_ref_part>`_.
```

Comment:
You can avoid Sphinx "duplicate reference" warnings by making the link "anonymous" with double underscore:
```suggestion
        Whether `ISR will be applied to the reference particle <https://impactx.readthedocs.io/en/latest/usage/python.html#impactx.ImpactX.isr_on_ref_part>`__.
```